static/img/deckgl_h3.html

```diff
@@ -31,10 +31,10 @@
     <div id="map"></div>
     <script>
       const { DeckGL, H3HexagonLayer } = deck;
-      const token = 'fsh_1uQkWaPFfB2O7Qy1zzOHS9'
+      const token = 'fsh_1uQkWaPFfB2O7Qy1zzOHS9' // todo: https://www.notion.so/fusedio/HTTP-endpoint-json-NDFrame-to_json-got-an-unexpected-keyword-argument-cls-189899d3b7638077add7e568bb1e0b98?pvs=4
```

Comment:
It seems this map is broken due to issue in Notion.

Reply:
https://www.notion.so/fusedio/HTTP-endpoint-json-NDFrame-to_json-got-an-unexpected-keyword-argument-cls-189899d3b7638077add7e568bb1e0b98?pvs=4

The udf output must be in `json` for the map to render H3.